types/node/module.d.ts

```diff
@@ -125,7 +125,7 @@ declare module "module" {
         interface ImportAttributes extends NodeJS.Dict<string> {
             type?: string | undefined;
         }
-        type ModuleFormat = "builtin" | "commonjs" | "json" | "module" | "wasm";
+        type ModuleFormat = "builtin" | "commonjs" | "json" | "module" | "module-typescript" | "wasm";
```

Comment:
Looking at https://github.com/nodejs/node/blob/main/lib/internal/modules/esm/formats.js, should `commonjs-typescript` also be on this list?

Reply:
You have right. I updated.